tests/antithesis/Makefile

```diff
@@ -75,3 +76,36 @@ validate-node-count:
 		echo "CFG_NODE_COUNT must be either 1 or 3 (got $(CFG_NODE_COUNT))"; \
 		exit 1; \
 	fi
+
+REQUIRED_K8S_TOOLS := kubectl kind kapp
+
+.PHONY: check-k8s-tools
+check-k8s-tools:
+	@for tool in $(REQUIRED_K8S_TOOLS); do \
+		if ! command -v $$tool >/dev/null 2>&1; then \
+			echo "Error: '$$tool' is missing.  Please install it before continuing." >&2; \
+			exit 1; \
+		fi \
+	done
+
+KIND_CLUSTER_NAME := $(shell grep name ./kind/cluster.yaml | sed 's/.*: //')
+
+.PHONY: antithesis-cleanup-local-k8s-cluster
+antithesis-cleanup-local-k8s-cluster: check-k8s-tools
+	if [[ $$(kind get clusters) =~ (^|[[:space:]])$(KIND_CLUSTER_NAME)($$|[[:space:]]) ]]; then \
```

Comment:
This is overdesigned. Just ask users to provide a working K8s cluster or run `kind create cluster`